.typos.toml

```diff
@@ -45,6 +45,7 @@ extend-exclude = [
 	"lib/SimplePie/",
 	"node_modules/",
 	"p/scripts/vendor/",
+	"p/themes/BlueLagoon/README.md",
```

Comment:
will not be necessary after merging #5694

Reply:
https://github.com/FreshRSS/FreshRSS/pull/5694/commits/2bde8f24d8e857c03d46a4aaa637f2593c0dfccd